epochStart/shardchain/trigger.go

```diff
@@ -441,8 +492,9 @@ func (t *trigger) checkIfTriggerCanBeActivated(hash string, metaHdr *block.MetaB
 	}
 
 	if metaHdr.IsStartOfEpochBlock() {
-		err := t.validatorInfoProcessor.ProcessMetaBlock(metaHdr, []byte(hash))
+		missingMiniblocks, err := t.validatorInfoProcessor.ProcessMetaBlock(metaHdr, []byte(hash))
```

Comment:
not only missing are here - but all of them.

Reply:
Now are only the missing ones